sw/device/lib/crypto/impl/ecdsa_p256/ecdsa_p256.h

```diff
@@ -98,10 +98,10 @@ otbn_error_t ecdsa_p256_sign(const ecdsa_p256_message_digest_t *digest,
  * @param result Buffer in which to store output (true iff signature is valid)
```

Comment:
```suggestion
 * @param result Buffer in which to store output (true if signature is valid)
```
Is this result argument really needed?

Reply:
Yes, the result argument is important and means something different than the `status_t` return value. If the signature fails verification but nothing "goes wrong" during the computation (e.g. hardware errors, failed preconditions), then we would expect an OK status but a `false` in the `result` buffer.

Reply:
Added some extra comment above to clarify this.